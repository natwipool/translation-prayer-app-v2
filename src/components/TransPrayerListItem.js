import React from 'react';

const TransPrayerListItem = ({ precept, category }) => (
  <div>
    <h3 className="list-item__title">{precept}</h3>
    <span className="list-item__sub-title">{category}</span>
  </div>
)

export default TransPrayerListItem;