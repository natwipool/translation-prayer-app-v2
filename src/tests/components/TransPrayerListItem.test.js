import React from 'react';
import { shallow } from 'enzyme';
import TransPrayerListItem from '../../components/TransPrayerListItem';
import transPrayers from '../fixtures/transPrayers';

test('should render TransPrayerListItem with a playlist', () => {
  const wrapper = shallow(<TransPrayerListItem {...transPrayers[1]}/>);
  expect(wrapper).toMatchSnapshot();
});