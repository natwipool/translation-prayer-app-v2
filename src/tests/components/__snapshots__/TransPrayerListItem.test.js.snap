// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TransPrayerListItem with a playlist 1`] = `
<div>
  <h3
    className="list-item__title"
  >
    อริยธนคาถา
  </h3>
  <span
    className="list-item__sub-title"
  >
    บทสวดมนต์ภาคเย็น
  </span>
</div>
`;
